alea/model.py

```diff
@@ -372,8 +378,8 @@ def _confidence_interval_checks(
     def confidence_interval(
             self, poi_name: str,
             parameter_interval_bounds: Tuple[float, float] = None,
-            confidence_level: float = None,
-            confidence_interval_kind: str = None,
+            confidence_level: Tuple[float] = None,
+            confidence_interval_kind: Tuple[str] = None,
```

Comment:
Why did you change these to tuple? I think they should just be float and str or not?

Reply:
Fixed, I don't know why I changed this.